<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userWindow</class>
 <widget class="QMainWindow" name="userWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>861</width>
      <height>401</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="page_vacant"/>
    <widget class="QWidget" name="page_buyer">
     <widget class="QTableWidget" name="buyer_table">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>531</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <widget class="QLineEdit" name="commodity_name">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>-20</x>
         <y>10</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;请输入商品名称：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>121</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;选中商品&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;  以购买&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_seller">
     <widget class="QTableWidget" name="seller_table">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>611</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <widget class="QLineEdit" name="commodity_ID">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;请输入商品ID：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_buyer_order">
     <widget class="QTableWidget" name="buyer_order_table">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>631</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;已购买订单：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_seller_order">
     <widget class="QTableWidget" name="seller_order_table">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>631</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#ffffff;&quot;&gt;已售出订单：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_information">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>241</width>
        <height>361</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="res.qrc">:/res/informationBackground.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; color:#ffffff;&quot;&gt;个人信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="name">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="phoneNumber">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="adress">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>210</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="balance">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>280</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>账户</string>
    </property>
    <addaction name="actioninformation"/>
    <addaction name="cancellation"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>买家</string>
    </property>
    <addaction name="actionbuyer"/>
    <addaction name="actionbuyer_order"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>卖家</string>
    </property>
    <addaction name="actionon_shelf"/>
    <addaction name="actionseller"/>
    <addaction name="actionseller_order"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="cancellation">
   <property name="text">
    <string>注销</string>
   </property>
  </action>
  <action name="actioninformation">
   <property name="text">
    <string>查看信息</string>
   </property>
  </action>
  <action name="actionchange_information">
   <property name="text">
    <string>修改信息</string>
   </property>
  </action>
  <action name="actionrecharge">
   <property name="text">
    <string>充值</string>
   </property>
  </action>
  <action name="actionbuyer">
   <property name="text">
    <string>浏览商品</string>
   </property>
  </action>
  <action name="actionseller">
   <property name="text">
    <string>已发布商品</string>
   </property>
  </action>
  <action name="actionbuyer_order">
   <property name="text">
    <string>已购买订单</string>
   </property>
  </action>
  <action name="actionseller_order">
   <property name="text">
    <string>已售出订单</string>
   </property>
  </action>
  <action name="actionon_shelf">
   <property name="text">
    <string>上架商品</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
